packages/transport/src/modules/sync/ChangedItems.ts

```diff
@@ -12,7 +12,8 @@ export class ChangedItems implements IChangedItems {
     public constructor(
         public readonly relationships: Relationship[] = [],
         public readonly messages: Message[] = [],
-        public readonly identityDeletionProcesses: IdentityDeletionProcess[] = []
+        public readonly identityDeletionProcesses: IdentityDeletionProcess[] = [],
+        public readonly changedObjectIdentifiersDuringDatawalletSync: string[] = []
```

Comment:
If you change it here, won't it need to be adjusted in the SyncEverythingResponse, too?

Reply:
Nope, I defenitely don't want to expose this.